tests/tasks/assert_certificate_parameters.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Verify certificate exists in the correct location
+  file:
+    path: "{{ cert['path'] }}"
+    state: file
+  register: result
+
+- name: Verify key exists in the correct location
+  file:
+    path: "{{ cert['key_path'] }}"
+    state: file
+  register: result
```

Comment:
do you need the register here?

Reply:
Same then https://github.com/linux-system-roles/certificate/pull/3#discussion_r433869562